wondrous-app/components/Settings/Members/index.tsx

```diff
@@ -236,6 +259,8 @@ function Members() {
           variables: {
             orgId,
             searchString: isSearchQueryENS ? walletAddress : searchQuery,
+            roleIds: selectedRoleIds,
+            limit: Number.POSITIVE_INFINITY,
```

Comment:
this is bad practice I believe, let's try and avoid using this and reuse the hasMore here since this can get pretty big for an org of 1000+ with most of the members having one main role ( contributor for example )

Reply:
done ser @Lamperoyge 🫡